configs,ext: Updated the gem5 SST Bridge to use SST 13.0.0 (#396)
gem5/gem5: ext/sst/INSTALL.md

### Downloading the SST-Core Source Code

```sh
wget https://github.com/sstsimulator/sst-core/releases/download/v13.0.0_Final/sstcore-13.0.0.tar.gz

> tar xvf sstcore-13.0.0.tar.gz

Reviewer: ```suggestion
tar xzf sstcore-13.0.0.tar.gz
```

Minor nits here:

1. `v` is just `verbose`. This is an uncesssary addition/
2. I like to be explicit and add `z` which means to also unzip (though it does do this automatically).

Author: Sure, I'll update this.

### Downloading the SST-Elements Source Code

```sh
wget https://github.com/sstsimulator/sst-elements/releases/download/v13.0.0_Final/sstelements-13.0.0.tar.gz

> tar xvf sstelements-13.0.0.tar.gz

Reviewer: ```suggestion
tar xzf sstelements-13.0.0.tar.gz
```

Author: Updated in the next patch.

### Building gem5 library

At the root of the gem5 folder, you need to compile gem5 as a library. This

> varies which OS you use. If you're using Linux, then type the following:

Reviewer: ```suggestion
varies  dependent on which OS you are useing If you're using Linux, then execute the following:
```

Author: done.

```sh
scons build/RISCV/libgem5_opt.so -j $(nproc) --without-tcmalloc --duplicate-sources
```

> In case you're using Mac, then type the following:

Reviewer: ```suggestion
If you are using Mac OS X, then execute the following:
```

```sh
scons build/RISCV/libgem5_opt.dylib -j $(nproc) --without-tcmalloc --duplicate-sources
```

**Note:**
* `--without-tcmalloc` is required to avoid a conflict with SST's malloc.
* `--duplicate-sources` is required as the compilation of SST depends on sources to be present in the "build" directory.
* The Mac version was tested on a Macbook Air with M2 processor.

### Compiling the SST integration

Go to the SST directory in the gem5 repo.
```sh
cd ext/sst
```

> According to the OS that you're using, you need to rename the `Makefile.xxx` to `Makefile`.

Reviewer: ```suggestion
Depending on your OS, you need to copy the correct `Makefile.xxx` file to `Makefile`.
```

```sh
cp Makefile.xxx Makefile    # linux or mac
make -j4
```

> Change `ARCH=RISCV` to `ARCH=ARM` in the `Makefile` in case you're compiling
> for ARM.

Reviewer: ```suggestion
The make file is hardcoded to RISC-V. IN the case you wish to compile to ARM, edit the Makefile by changing `ARCH=RISCV` to `ARCH=ARM`.
```
